src/CodeGen_AMDGPU_Dev.cpp

```diff
@@ -0,0 +1,507 @@
+#include "CodeGen_AMDGPU_Dev.h"
+#include "CodeGen_Internal.h"
+#include "ExprUsesVar.h"
+#include "IREquality.h"
+#include "IROperator.h"
+#include "IRPrinter.h"
+#include "IRMatch.h"
+#include "IRMutator.h"
+#include "Debug.h"
+#include "Simplify.h"
+#include "Solve.h"
+#include "Target.h"
+#include "LLVM_Headers.h"
+#include "LLVM_Runtime_Linker.h"
+
+#include <fstream>
+
+/*
+// This is declared in NVAMDGPU.h, which is not exported. Ugly, but seems better than
```

Comment:
I assume this comment is not accurate :)

Reply:
Yes, it is not accurate. I left it there so that you can provide more information about the purpose of `createNVVMReflectPass`

Reply:
It's some weird extra pass required by the nvptx backend in llvm. I would assume it's not relevant here.